node/tests/hubController-testConfig.json

```diff
@@ -4,20 +4,24 @@
         "opent2tBlob" : {
             "schema":"opent2t.p.light",
             "translator":"opent2t-translator-com-wink-lightbulb",
-            "controlId":"1985159"
+            "controlId":"2120684"
         }
     }, 
     "setResource": { 
         "opent2tBlob" : {
             "schema":"opent2t.p.light",
             "translator":"opent2t-translator-com-wink-lightbulb",
-            "controlId":"1985159"
+            "controlId":"2120684"
         },
         "deviceId": "",
         "resource": "power",
         "resourceBlob": {"value":true}
     },
     "subscription": {
+        "subscriptionInfo": {
+            "callbackUrl": "http://drsubscriptiontest.azurewebsites.net/subs",
```

Comment:
you know if we use compute instead of websites, we can actually use their azure emulators for local testing -- just a thought as we try to scale this for testing.